TwitchDownloaderWPF/PageChatRender.xaml.cs

```diff
@@ -295,11 +298,19 @@ public void SaveSettings()
 
         private bool ValidateInputs()
         {
-            if (!File.Exists(textJson.Text))
+            if (fileNames.Length == 0)
             {
-                AppendLog("ERROR: JSON File Not Found");
+                AppendLog("ERROR: no JSON Files selected");
                 return false;
             }
+            foreach (string fileName in fileNames)
+            {
+                if (!File.Exists(fileName))
+                {
+                    AppendLog("ERROR: JSON File Not Found");
```

Comment:
It should report the path since there is more than one possible missing file

Reply:
This is just whitespace change, nothing I added (aside from checking every file in a loop rather than a single file).